tests/dump.txt

```diff
@@ -0,0 +1,63 @@
+digraph "lib_spdif_tests" {
```

Comment:
Is this file supposed to be in here? I don't know what it is or where it's used.